css/amp.css

```diff
@@ -862,8 +862,8 @@ amp-story .i-amphtml-loader {
   display: none !important;
 }
 
-amp-story-page:not(:first-of-type):not([distance]):not([active]) {
-  transform: translateY(1000vh) !important;
+amp-story-page:not(:first-of-type) {
+  transform: translateY(1000vh);
```

Comment:
Maybe we should keep the `!important`, since publishers could have transforms in their styles